src/plugins/intel_cpu/src/nodes/llm_mlp.cpp

```diff
@@ -130,15 +147,81 @@ class Linear {
             }
         });
     }
+};
+
+static GateUpCombine jit_gateup_silu(dnnl_eltwise_swish);
+static GateUpCombine jit_gateup_gelu(dnnl_eltwise_gelu_tanh);
+
+class LinearGateUp {
+public:
+    std::vector<Work> works;
+
+    int used_nthr = 0;
+
+    LinearGateUp() {}
+
+    WeightBuffer wbuffer;
+
+    // weight [N, K]
+    // Gate & Up are interleaved in N dimension: 16-gate / 16-up
+    // and post-ops will compute  silu(gate)*up in unit of 16 elements
+    // and store out as bfloat16.
+    template <typename T>
+    void setup(T* p_weight_gate, T* p_weight_up, int stride, int N, int K) {
+        const int blk_K_size = 256;
+        // prepare weights, split N among threads
+        // in unit of 32
```

Comment:
can we use meaningful alias for 32 here and below? Like NBlock or similar.

Reply:
Sure, changed.